### ISSUE TITLE
bug(application): Uncaught TypeError: Cannot match against 'undefined' or 'null'.

### ISSUE THREAD

**Issue by cfobel** [sci-bots/microdrop-3.0#19]
As of c8d95beab7d5f2566231f4884e3a5cf0a9d57366, the Electron app fails to launch with the following error:

![image](https://user-images.githubusercontent.com/823355/36029098-c2c41eda-0d6f-11e8-9ec0-4eb37a1ba65e.png)

Console log is:

```sh
(md-3-dev) PS C:\Users\chris\Miniconda2-x86\conda-bld\microdrop-3_1518178803860\work\microdrop-builder\packager\Microdrop-win32-ia32> .\Microdrop.exe
(md-3-dev) PS C:\Users\chris\Miniconda2-x86\conda-bld\microdrop-3_1518178803860\work\microdrop-builder\packager\Microdrop-win32-ia32>
Uncaught TypeError: Cannot match against 'undefined' or 'null'.
Uncaught TypeError: Cannot match against 'undefined' or 'null'.
Uncaught TypeError: Cannot match against 'undefined' or 'null'.
Uncaught TypeError: Cannot read property 'mqtt_ws_port' of null
```

**Comment by cfobel** [comment]
Maybe this has to do with launching through `electron .\node_modules\@microdrop\application\bin\microdrop` instead of `electron .\node_modules\@microdrop\application` ?

Is there a reason why the code in `.\node_modules\@microdrop\application\bin\microdrop` cannot just be moved into `.\node_modules\@microdrop\application\index.js`?

**Comment by Lucaszw** [comment]
This is fixed as of b5b67572133e72553df8e6c5b8de2b2a05b801f4. Closing
